stonesoup/proposal/tests/test_simple.py

```diff
@@ -0,0 +1,132 @@
+import itertools
+
+import datetime
+import numpy as np
+
+# Import the proposals
+from stonesoup.proposal.simple import PriorAsProposal, KFasProposal
+from stonesoup.models.transition.linear import ConstantVelocity
+from stonesoup.types.particle import Particle
+from stonesoup.types.prediction import ParticleStatePrediction
+from stonesoup.predictor.kalman import KalmanPredictor
+from stonesoup.updater.kalman import KalmanUpdater
+from stonesoup.types.state import ParticleState, GaussianState
+from stonesoup.predictor.particle import ParticlePredictor
+from stonesoup.types.detection import Detection
+from stonesoup.models.measurement.linear import LinearGaussian
+from stonesoup.types.hypothesis import SingleHypothesis
+
+
+def test_prior_proposal():
+    # test that the proposal as prior and basic PF implementation
+    # yield same results, since they are driven by the transition model
+    np.random.seed(16549)
```

Comment:
See #1064 which aims to remove any setting of global random seeds in tests.

May need to add random state option to Proposal to support this, like models have.